src/Components/PropertyInspector/PropertyInspector.scss

```diff
@@ -2,11 +2,15 @@
 
 .cb-property-inspector-container {
     height: 100%;
-    padding-left: 20px;
+    padding: 8px 20px;
```

Comment:
is this a class only used by your flyout?

Reply:
This class is used in the normal property inspector. Are we sure this doesn't have any impact where we use it today?

Reply:
Ok. so then let's remove this padding on top then cause it's pushing the inspector down now. 
![image](https://user-images.githubusercontent.com/57726991/185977431-bca3cb5e-421f-4323-a92d-0630d1b89e7a.png)

Reply:
we will also need to make sure this still looks good in the ADT Explorer since that's primarily where this control is used.